python/lsst/cp/verify/mergeResults.py

```diff
@@ -477,6 +479,172 @@ class CpVerifyRunMergeTask(CpVerifyExpMergeTask):
 # End ExpMerge/RunMerge
 
 
+class CpVerifyDetMergeConnections(pipeBase.PipelineTaskConnections,
+                                  dimensions={"instrument", "detector"},
+                                  defaultTemplates={}):
+    inputStats = cT.Input(
+        name="detectorStats",
+        doc="Input statistics to merge.",
+        storageClass="StructuredDataDict",
+        dimensions=["instrument", "exposure", "detector"],
+        multiple=True,
+    )
+    inputResults = cT.Input(
+        name="detectorResults",
+        doc="Input results to merge.",
+        storageClass="ArrowAstropy",
+        dimensions=["instrument", "exposure", "detector"],
+        multiple=True,
+    )
+    inputMatrix = cT.Input(
+        name="detectorMatrix",
+        doc="Input matrix to merge.",
+        storageClass="ArrowAstropy",
+        dimensions=["instrument", "exposure", "detector"],
+        multiple=True,
+    )
+    camera = cT.PrerequisiteInput(
+        name="camera",
+        storageClass="Camera",
+        doc="Input camera.",
+        dimensions=["instrument", ],
+        isCalibration=True,
+    )
+
+    outputStats = cT.Output(
+        name="exposureStats",
+        doc="Output statistics.",
+        storageClass="StructuredDataDict",
+        dimensions=["instrument", "exposure"],
```

Comment:
This, and the other outputs, should have dimensions `["instrument", "detector"]`.